apps/user/apps.py

```diff
@@ -3,3 +3,6 @@
 
 class UserConfig(AppConfig):
     name = 'user'
+
+    def ready(self):
+        import user.receivers  # noqa
```

Comment:
why though?

Reply:
Otherwise, Receivers files won't work. If receivers are written in models.py, then they are loaded while app is loaded. But if separated from models.py into other file, the file is not loaded initially so receivers won't work.

Reply:
Oh, good good